src/Cameras/VR/vrExperienceHelper.ts

```diff
@@ -844,18 +844,18 @@ export class VRExperienceHelper {
                         });
 
                         this.xr.input.onControllerAddedObservable.add((controller) => {
-                            var webVRController = controller.gamepadController;
-                            if (webVRController) {
-                                var localController = new VRExperienceHelperControllerGazer(webVRController, this._scene, this._cameraGazer._gazeTracker);
-
-                                if (controller.inputSource.handedness === "right" || (this._leftController && this._leftController.webVRController != webVRController)) {
-                                    this._rightController = localController;
-                                } else {
-                                    this._leftController = localController;
-                                }
-
-                                this._tryEnableInteractionOnController(localController);
-                            }
+                            // var webVRController = controller.gamepadController;

```

Comment:
Comments?

Reply:
Commented on that in the description - (temporarily) removed gaze support in the vr helper when XR is used. This will return when teleportation will move to the feature manager

Reply:
Makes sense!

Reply:
(when this is closed - https://github.com/BabylonJS/Babylon.js/issues/7290)
The main issue is that the webxr motion controller is no longer a gamepad object, so the old gaze suddenly doesnt support it.